Cargo.toml

```diff
@@ -18,7 +18,7 @@ tempfile = "3.3.0"
 lazy_static = "1.4.0"
 openssl = "0.10.60"
 sha2 = "0.10.6"
-meesign-crypto = { git = "https://github.com/crocs-muni/meesign-crypto", rev = "92fbdc7", default-features = false }
```

Comment:
Please, change it to the repository again. The changes got merged (https://github.com/crocs-muni/meesign-crypto/pull/3).

Reply:
Done.